frontend/hub/collections/CollectionPage/CollectionDocumentation.tsx

```diff
@@ -53,18 +118,65 @@ export function CollectionDocumentation() {
             <CollectionDocumentationTabPanel
               setDrawerOpen={setDrawerOpen}
               groups={groups}
-              content={content}
-              setContent={setContent}
+              setSearchText={setSearchText}
             />
           ) : undefined
         }
       >
-        <DrawerContentBody>
-          <CollectionDocumentationTabContent
-            content={content}
-            isDrawerOpen={isDrawerOpen}
-            setDrawerOpen={setDrawerOpen}
-          />
+        <DrawerContentBody className="body hub-docs-content pf-v5-c-content hub-content-alert-fix">
+          {/* This enables scrolling using keyboard after click into the content*/}
+          <div
+            style={{ outline: 'none' }}
+            tabIndex={0}
+            onClick={(event) => {
+              event?.currentTarget?.focus();
+            }}
+            /* listener and role needed for eslint */
+            onKeyDown={() => {}}
+            role="button"
+          >
+            {content && (
+              <>
+                <PageSection variant="light" id="Title_part">
+                  <Title headingLevel="h1">
+                    {content?.content_type + ' > ' + content?.content_name}
+                  </Title>
+
+                  {/* Checkbox allows switch between rendered docs in HTML and between plain JSON that displays raw documentation data*/}
+                  <>
+                    <Checkbox
+                      onChange={(event, checked) => setRenderJson(checked)}
+                      isChecked={renderJson}
+                      id="render-json-checkbox"
+                    />{' '}
+                    {t('Render documentation as JSON')}
```

Comment:
Hey @MilanPospisil I could not find this feature in the old app.


<img width="1434" alt="image" src="https://github.com/ansible/ansible-ui/assets/9053044/d069d39e-8dd3-4525-91f8-4ebfb845b585">

Old App

![image](https://github.com/ansible/ansible-ui/assets/9053044/248629ff-bce9-4372-ba37-5337e3267ddb)





 Render documentation as JSON

Reply:
It is not, I must discuss it with UX, because sometimes it can happen that collection is not rendered correctly. In those cases, only source can be the JSON. And Im rendering JSON also when rendering failed.

Reply:
Anastasia Ratti told me that this is ok, she only wanted to change checkbox to ToggleGroup.